test/helpers/transactions.ts

```diff
@@ -87,25 +88,29 @@ export const createTransaction = async (
   options: TransactionOptions,
   txType?: EthTransactionType
 ): Promise<string> => {
-  const defaultTxnStyle = MoonwallContext.getContext()!.defaultEthTxnStyle;
-
-  const isLegacy = txType
-    ? txType === "legacy"
-    : defaultTxnStyle
-    ? defaultTxnStyle === "legacy"
-    : true;
-
-  const isEip2930 = txType
-    ? txType === "eip2930"
-    : defaultTxnStyle
-    ? defaultTxnStyle === "eip2930"
-    : true;
-
-  const isEip1559 = txType
-    ? txType === "eip1559"
-    : defaultTxnStyle
-    ? defaultTxnStyle === "eip1559"
-    : true;
+  const defaultTxnStyle = "legacy";
+
+  const isLegacy = true;
```

Comment:
I don't understand why we are forcing to be legacy all the time no matter the transactionType that is given

Reply:
Ported legacy code, have axed the whole lot since moonwall has much nicer ways of creating transactions.